docs/variants/talos_2/building-manual.md

```diff
@@ -23,13 +23,20 @@ To build coreboot image, follow the steps below:
     docker run --rm -it \
        -v $PWD:/home/coreboot/coreboot \
        -w /home/coreboot/coreboot \
-       3mdeb/coreboot-sdk:mkimage /bin/bash
+       -u "$(id -u):$(id -g)" \
+       coreboot/coreboot-sdk:0ad5fbd48d /bin/bash
     ```
 
 1. Inside of the container, configure and start the build process:
 
     ```bash
-    (docker)cp configs/config.raptor-cs-talos-2 .config
+    (docker)cat > .config <<EOF
+    CONFIG_VENDOR_RAPTOR_CS=y
+    CONFIG_TALOS_2_INFINEON_TPM_1=y
+    CONFIG_TPM_LOG_TPM2=y
+    CONFIG_PAYLOAD_SKIBOOT=y
+    CONFIG_LOCALVERSION="v0.7.0"
```

Comment:
@SergiiDmytruk @krystian-hebel  Why we do not want to include these options in the config in the code tree?

Reply:
I agree, this should be in the tree.

Reply:
@krystian-hebel This is what I got, after following documentation and `make savedefconfig`: https://github.com/Dasharo/coreboot/commit/fc47236e9877f4113dfcce07fa928f52d4d2c8ee

Reply:
@macpijan looks correct, but please check if `make olddefconfig` from this file works. I had some strange issues around TPM options in `menuconfig`, I hope they don't apply here.

Reply:
Works and builds